Tasks/HybridFlowProcessor.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using Tasks.DoNotChange;
+
+namespace Tasks
+{
+    public class HybridFlowProcessor<T> : IHybridFlowProcessor<T>
```

Comment:
Tests passed, but:
I think they mean by reuse code, is that under the hood you should use type of DoublyLinkedList  as storage